src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Zen+Antique+Soft&display=swap");
```

Comment:
it is better to add fonts via a link on the <head>
https://mate.academy/ru/learn/css-basics/colors-and-fonts/theory?section=practice